### PR TITLE
Split up views into individual files

### LINKED ISSUE #40: Split views into multiple files
Now that lots of different features and views are being added, it's probably not the best idea to keep every view in a single file that's going to end up being hundreds and hundreds of lines long. One way to do this would be with Flask [Blueprints](http://flask.pocoo.org/docs/0.10/blueprints/).

### PR DESCRIPTION
**Pull request by MaraJade**
Fixes #40 
## Changes in this PR.

- [X] Put views into individual files in the views directory
- [X] Adjusted views and tests to reflect the move
## Testing this PR.

1. Run `nosetests`
### Expected Output.

```
$ nosetests
............................
----------------------------------------------------------------------
Ran 28 tests in 0.236s

OK
```

@osuosl/devs

### CODE REVIEW

**Comment by Kennric** [comment]
Works for me. +1

**Comment by aaroncohen73** [comment]
I'm not sure that we should be putting is_logged_in and get_user into their own files inside the `views/`
 because they aren't views. I think that it might make more sense to put those functions inside a file called something along the lines of `util.py` inside the `app/` directory.

File: app/__init__.py [hunk 1]
```diff
@@ -3,5 +3,12 @@
...
 
-from app import views  # NOQA flake8 ignore
+from app.views import index  # NOQA flake8 ignore
+from app.views import admin  # NOQA flake8 ignore
+from app.views import create_time  # NOQA flake8 ignore
+from app.views import get_user  # NOQA flake8 ignore
+from app.views import is_logged_in  # NOQA flake8 ignore
+from app.views import login  # NOQA flake8 ignore
+from app.views import logout  # NOQA flake8 ignore
+from app.views import view_times  # NOQA flake8 ignore
```
Comment (pop) [inline app/__init__.py:16]: Do these need to be on individual lines?

File: app/views/create_time.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        activities = form.activities.data
+        notes = form.notes.data
+        issue_uri = form.issue_uri.data
+
+        # With project name, get slug
+        for project in projects:
+            if project['name'] == project_name:
+                project_name = project['slugs'][0]
+
+        # Try to convert str to int, else just leave as str
```
Comment (ElijahCaine) [inline app/views/create_time.py]: Why?

**Comment by pop** [comment]
Small nitpick. +1 from me once that's addressed.

**Comment by aaroncohen73** [comment]
+1

**Comment by pop** [comment]
+1

**Pull request merged**